Assets/Photon Unity Networking/Resources/PhotonServerSettings.asset

```diff
@@ -15,14 +15,14 @@ MonoBehaviour:
   AppID: 
   VoiceAppID: 
   ChatAppID: 
-  HostType: 2
```

Comment:
Should undo these changed to the PhotonServerSettings as merging these into development will break online functionality.